crates/ra_cli/src/analysis_bench.rs

```diff
@@ -84,7 +85,27 @@ fn do_work<F: Fn(&Analysis) -> T, T>(host: &AnalysisHost, work: F) -> T {
     {
         let start = Instant::now();
         eprint!("trivial change: ");
-        host.raw_database().salsa_runtime().next_revision();
+        host.raw_database().salsa_runtime().synthetic_write(Durability::LOW);
+        work(&host.analysis());
+        eprintln!("{:?}", start.elapsed());
+    }
+    {
+        let start = Instant::now();
+        eprint!("comment change: ");
+        {
+            let mut text = host.analysis().file_text(file_id).to_string();
+            text.push_str("\n/* Hello world */\n");
```

Comment:
a slightly more interesting change might be putting the comment at the top?

Reply:
That would be harder to implement, because I would need to adjust offset for subsequent queries. 

ideally, we should accept a `.patch` file as a command line argument...